python/test/unit/mesh/test_mesh_utils.py

```diff
@@ -104,4 +111,57 @@ def test_partition():
     boundary = cpp.mesh.compute_interior_facets(topology)
     topology.set_interior_facets(boundary)
     boundary = topology.on_boundary(topology.dim - 1)
-    print(boundary)
+    # print(boundary)
+
+    # Build distributed cell-vertex AdjacencyList and IndexMap for
+    # vertices
+
+    print(topology.connectivity(2, 0).array())
+    print(global_to_local_vertices)
+    # return
+    cells, vertex_map, l2g = cpp.mesh.create_distributed_adjacency_list(cpp.MPI.comm_world, topology,
+                                                                        global_to_local_vertices)
+
+    print("l2G", l2g)
+    print("cells: ", cells.array())
+    print("offset: ", cells.offsets())
+    return
```

Comment:
Forgotten here? Also I am not sure if you wanted to leave all those prints here. Running `pytest . -s` is then too poluted.